docs/source/pythonapi/examples/mgxs-part-iv.ipynb

```diff
@@ -1076,13 +1072,83 @@
fH8ieVGP\nh0jT9gCm5fqwbsRQYH/gGuBrwoccwIPAtsDNhEceuwCXmdlq7n4ygJltFB03CTgH+AlYF/jfB76Z\nHQH8mfCBfiXQidBdvwWhd6YhHwHtzexQd7+9sW/AzHYAHiaMe/g/wjiMXwNPmtkgd381Oq4X8Aqw\nPHA98DawOrAf0AUYD1wFjAL+SHi8AzA1eq3XExKNZ3ga+DnwF8LPbn/gVjPrlpnkRQ4mfGhfF13r\nNOA+M+vt7osa+PbuA74l/Ozuir7PH6L7941ing1cAiwEfkdImoa4+ytZ17oW+BI4j5CgNuQ5Qk/E\nICCdaG0NvOPur5nZp4RkY2K0b1D0+nzGNQ4AlgGuBmYSEpcTgF7RzwHC71wvwu/ZodE9M8fQ3ALU\nRK9XEpLRUUB/Mxuc7nEh/Cz7AncSfrbXs+TfTCQ5d9emTVsDG7Ac4X/a98c4ZzGwANggq32vaN/p\nWe3/IHy4rRN9fQKwCFixkXs8AExO8P30AGZEcbxJ+NA8CFg+x7FvAw9ltS0DvA88ktF2W/T9btrI\nffeNvqchOfY9BTyZ8XX6+z8oo60d4QN4NtA1alsr+j6+zIyfMFZlEfCrJn4W6fNPyvGznQusldG2\nanTvpzLaDovOf5povFweP/8vgMcyvv4PcFP0/m7g7ox9LwNTs3/+Oa55VvT70yuj7a/A/BzHbhfF\nvG9W+25R+34ZbZ9EP8ftCvnflDZtetQi0rh09/P3Mc972t3fzmrbjfABkf0X+58Ijz13i76eFb3u\nHY0PyWUWsEZTjxSyuftXhJ6RvwIrEP6Svwv40szOTh9nZpsA6wG1ZtY9vRESsSeA9GMSIyRUY919\nIoWxG/CFu/+v58ZDz8VVhJ6NbbOOv9vdv8v4+lnCX/m9497YzKqAnYAH3P2jjPt/Qfg5DTazZTNO\nceBGd893lP4LwBZmVhX97LaM2iAkVttEcXQGNiHjMUsUx//GCUWP7bpH51t0fFP2I/SUPJ317/oq\nIdnaPuv499z96Ty/N5G8KPEQaVz6A225mOd9mKNtLeAzd/8xq31qxn4IPSDPAzcCM8ys1sz2z0pC\nLiU8GnjZzN4xs6tzjb3Ixd1nuPux7r4asAGhm/1L4DwzSz/DXy96vR34KmP7kjBAtaOFwag9CMnZ\nG/ncO09rAe/maJ9K+IBdK6u93rgKd08nbismuHcPwuOhdxq5/5pZ7R/GuP5zLBnLsTHhZ5d+lPIC\nsJqFKcdbEh6F10s8onEtt5vZN4R//68IiSBAtzzuvx6wEvX/Tb8i9IJ1AlbJOn4aIgWmMR4ijXD3\n783sM+AXMU/NNTukod6Len8te5g1MMTMtifMltkVOBB4wsx29uAtM9uAMP5kV2Af4BgzO8/dz8s3\nSHd/D3jPzB4mfNgfTHj2n/6j5GTgtQZO/4HwYVVocWdkNDSOI8nMjiTnxJkJlDnOYwEwM6NnbFJ0\nrUGE3prsgaXtgHGEJPgiwqOwOcDPqP9v1pgq4DNCzZJc32v24OlizXKSNkyJh0jT/g0cYWZbeLwB\nptk+BHYws65ZvR4bRa8fZR7s7k8Rxj+cYmZnEAZmbk80MNHd5xJqY/zTQi2MB4CzzOxiz5qd0hR3\n/8DMviUMSoQwjgPge3d/soHTMLMvCb1CGzd1ixjhfEjuRK9P9PpRjn2F8iXhw3yDBu7vND5zpSkT\nousPAuYDL6Z3uPsiCzU9BgHrEOp3vJ9x7iaEAbc17v6PdKOZ7ZrjPg39vN8HBgPPufuCZnwfIonp\nUYtI00YTPixuMrPsrmjM7Odmdnwe13mYkOwfl9WeIgzs+090vVyPCF4j/IW6THTMSpk7PdTCmEr4\nb7pDQwGY2S8tR3VSM9ucUEgsPeOkjvAhdYqZLTVTw8xWju7rwL+APc1sQEP3BX6M4l+hkWPSHgZW\nNbMDM+7XjvBI6HvgmTyukYiHGR2PEWpq/K/Kqpn1JMwEGe/uPzTj+gsJg0YHEWa0vJB1yAuE8TNb\nkvWYhSU9O//7/3b0+O0Elk40fiRMp83+t76HUPju7Kx2LEx9Xj67XaTQ1OMh0gR3n2ZmwwmzDqaa\nWWbl0q0JUz3/lsd1xlqoV3GhmfVmyXTaPYEx7v5BdOgfLNS4eIjw131P4GjgY5Z8GD1mZl8QxgfM\nIPSaHAs8mGMMSaZDgIPN7AFCcjE/OvfXhG71i6NY3cx+S0gC3jCzvwHTCdNktyfM8NgruuaZhAGZ\n483sBkICtBphIOM20cDPSYQPztPMbAXCFOEn3P3rHDHeQBj0ems0ePZDws94K+CEJr6/Qjgb2BF4\n3syujeI+kvDvfWrWsUkezTxHmF3i1J8qCyHxOIOsxyyRNwhF7K40s7UIj7r2Y8kA6Ex10evVZjYO\nWODu/3T3J83sZuDsKFEcRxjwvH50raOBsQm+J5H8lXtaTTE2QmXGpwj/oU4iY4qYNm1JN0I393WE\nnoC5hJkl4wn/s+6Qcdwi4M8NXKMLoVjUJ4TCYm8BqaxjtgPuj46ZG73eAfw845jfRr/j6UcD7xCS\nhmWb+B76EmpTvEIYVPgT8ClQC/TPcXw/wuOcL6NYpkXHbpd13BqE5OuLKJ53CXVG2mccMzJqn0/G\n1Nro+3gi63orAzcRkqq50X/Hh2Qds1Z0nVSOuBcB5zTxs2js/P6EpGs2oZflceCXWcccFp0/IObv\n0U7RefOATln7Voz2LQQG5ji3TxTLd9HP+poo1kXA8Izjqgizp2ZE15qfdZ0jot+BHwj1TCYCFwKr\nZBzzMXBfuf+709b6tla5VouFUsOruPvkqIu0DljPwzNxERERKZNWOcbD3b9w98nR+xmEypErNX6W\niIiIFFurTDwymdlAoMrdp5c7FhERkbau4hIPMxtsZmMtLDa12MyqcxxzrJl9YGHxqP9GI/JzXWsl\nQjnnI4odt4iIiDSt4hIPwiJLkw
```

Comment:
Its difficult to tell what line to add a space to.  Do you mean between the CE and MG plots the first time through?  If so I just did.